geoserver/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# A GeoServer installation
+FROM java:8-alpine
+
+MAINTAINER Azavea
+
+ENV GEOSERVER_VERSION 2.11.2
+ENV GEOSERVER_ZIP_URL http://sourceforge.net/projects/geoserver/files/GeoServer/${GEOSERVER_VERSION}/geoserver-${GEOSERVER_VERSION}-bin.zip
+
+ENV GEOSERVER_DIR /usr/share
+ENV GEOSERVER_EXT_DIR $GEOSERVER_DIR/geoserver-$GEOSERVER_VERSION/webapps/geoserver/WEB-INF/lib
+ENV PATH /usr/share/geoserver-$GEOSERVER_VERSION/bin:$PATH
+ENV JAVA_HOME /usr/lib/jvm/java-1.8-openjdk
+ENV GEOSERVER_HOME $GEOSERVER_DIR/geoserver-$GEOSERVER_VERSION/
+
+RUN apk add --update ca-certificates openssl && update-ca-certificates && \
+  wget -O $GEOSERVER_DIR/geoserver.zip $GEOSERVER_ZIP_URL && \
+  unzip -qq $GEOSERVER_DIR/geoserver.zip -d $GEOSERVER_DIR
+
+COPY data_dir /data
```

Comment:
I _think_ it's going to go away. I misread intent for this directory. Geoserver needs to know where to put its data, but we don't need to see it or care about it I think. I was under the impression originally that we'd be placing thing there directly.

Reply:
This should probably be initialized with some data to start with that gets Geoserver in a running state. In [development](https://github.com/PublicMapping/DistrictBuilder/pull/464/files#diff-4e5e90c6228fd48698d074241c2ba760R45) those files get mounted directly. It might make sense to include some of those files in this PR if I understand correctly.